pkg/vm/vm_test.go

```diff
@@ -346,34 +348,168 @@ func TestInferOperandSub(t *testing.T) {
 	assert.Equal(t, expectedOp0Cell, op0Cell)
 }
 
-func TestComputeAddRes(t *testing.T) {
+func TestComputeRes_Unconstrained(t *testing.T) {
 	vm := defaultVirtualMachine()
+	instruction := Instruction{Res: Unconstrained}
 
-	instruction := Instruction{
-		Res: AddOperands,
-	}
+	cellOp0 := &mem.Cell{Accessed: true, Value: mem.MemoryValueFromInt(10)}
+	cellOp1 := &mem.Cell{Accessed: true, Value: mem.MemoryValueFromInt(15)}
 
-	cellOp0 := &mem.Cell{
-		Accessed: true,
-		Value: mem.MemoryValueFromMemoryAddress(
-			mem.NewMemoryAddress(2, 10),
-		),
-	}
+	res, err := vm.computeRes(&instruction, cellOp0, cellOp1)
+	require.NoError(t, err)
+	assert.Nil(t, res)
+}
 
-	cellOp1 := &mem.Cell{
-		Accessed: true,
-		Value:    mem.MemoryValueFromInt(15),
-	}
+func TestComputeRes_Op1(t *testing.T) {
+	vm := defaultVirtualMachine()
+	instruction := Instruction{Res: Op1}
+
+	cellOp0 := &mem.Cell{Accessed: true, Value: mem.MemoryValueFromInt(10)}
```

Comment:
`cellOp0` is not used in this test, so no need to create it to test the function

Reply:
done. Cell Operand 0 set to nil, the test succeeds.